tests/integration/Owncloud/OcisSdkPhp/OcisTest.php

```diff
@@ -138,15 +138,43 @@ public function testCreateDriveInvalidQuota(int $quota): void
         // no new drive should have been created
         $this->assertCount($countDrivesAtStart, $ocis->listMyDrives());
     }
+    /**
+     * @return array<int, array<int, string>>
+     */
+    public function groupMockData(): array
+    {
+        return[
+            ["philosophy-haters","sda"],
+            ["physics-lovers","sadas"]
+        ];
+    }
+
+    /**
+     * @dataProvider groupMockData
+     *
+     * @param string $groupName
+     * @param string $groupDescription
+     * @return void
+     */
+    public function testCreateGroup($groupName, $groupDescription)
+    {
+        $token = $this->getAccessToken('admin', 'admin');
+        $ocis = new Ocis($this->ocisUrl, $token, ['verify' => false]);
+        $ocis->createGroup($groupName, $groupDescription);
+        $this->markTestSkipped('https://github.com/owncloud/ocis-php-sdk/issues/67');
+        // $this->assertInstanceOf(Group::class, $newGroup);
+        // $this->assertSame($newGroup->getDisplayName(),$groupName);
+        // $this->assertSame($newGroup->getDescription(),$groupDescription);
```

Comment:
that should not be a problem, because oDataError is not catch in the createGroup function